test/runtests.jl

```diff
@@ -9,6 +9,9 @@ using PowerSystems
 import PowerSystems: PowerSystemTableData
 const PSY = PowerSystems
 
+include(joinpath(@__DIR__, "../src/utils/data.jl"))
+import .UtilsData: TestData
+download(TestData; branch = "master")
```

Comment:
This is already called about 10 lines down.

Reply:
removed